### PR TITLE
chore: improve price parameter names to clarify usage.

### PR DESCRIPTION
**Pull request by acompa**
As suggested via user feedback.

### CODE REVIEW

File: notdiamond/llms/config.py [hunk 1]
```diff
@@ -55,8 +55,11 @@ def __init__(
         system_prompt: Optional[str] = None,
         context_length: Optional[int] = None,
         input_price: Optional[float] = None,
```
Comment (r0ymanesco) [inline notdiamond/llms/config.py:57]: Why are we keeping the old parameters if you're renaming them to `custom_*`?

Comment (acompa) [reply notdiamond/llms/config.py:57]: Backward compatibility

Comment (acompa) [reply notdiamond/llms/config.py:57]: Note that they're kept, but undocumented, and the default is to check the `custom_*` variants

Comment (r0ymanesco) [reply notdiamond/llms/config.py:57]: Ok makes sense

**Review by r0ymanesco — APPROVED** [review]

File: notdiamond/llms/config.py (hunk 1 above)
Comment (r0ymanesco) [reply notdiamond/llms/config.py:57]: When you merge this, can you also update the docs too?

**Comment by acompa** [comment]
Merging as failures are due to o1 model issues. Will fix separately.

**Pull request merged**